collector-exoplayer/src/main/java/com/bitmovin/analytics/exoplayer/ExoUtil.java

```diff
@@ -43,4 +45,11 @@ public static String getUserAgent(Context context) {
     }
     return com.google.android.exoplayer2.util.Util.getUserAgent(context, applicationName);
   }
+
+  public static boolean getIsLiveFromConfigOrPlayer(boolean isPlayerReady, CollectorConfig config, boolean isLiveFromPlayer) {
```

Comment:
Why do we have two utils that do the same thing? 
We could move this to a util function in `Collector`?

Reply:
moved method and tests to collector